helm/alfresco-process-infrastructure/values.yaml

```diff
@@ -706,6 +706,8 @@ alfresco-deployment-service:
     {{- end }}
     - name: ATS_TRANSFORMER_TIKA_URL
       value: "http://tika.{{ .Release.Namespace }}/transform"
+    - name: PROCESS_UI_URL
+      value: '{{ template "alfresco-process-infrastructure.acs-url" . }}'
```

Comment:
@eromano @mteodori Can you guys come up with a more sensible default value?

Reply:
common.gateway-url